### PR TITLE
fix rbac issues

### PR DESCRIPTION
**Pull request by 2005wind**
fix rbac issues . because the k9s name do not support space .

### CODE REVIEW

**Comment by wu-sheng** [comment]
Could you describe the issue and your use case?

File: chart/operator/templates/rbac.yaml [hunk 1]
```diff
@@ -473,7 +473,7 @@ subjects:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: {{ include "operator.fullname" . | trunc (int (sub 63 (len "-manager-role"))) | printf "%s-manager-role" }} binding
+  name: {{ include "operator.fullname" . | trunc (int (sub 63 (len "-manager-role"))) | printf "%s-manager-role" }}binding
```
Comment (kezhenxu94) [inline chart/operator/templates/rbac.yaml:476]: Let's add a dash here 

```suggestion
  name: {{ include "operator.fullname" . | trunc (int (sub 63 (len "-manager-role"))) | printf "%s-manager-role" }}-binding
```

File: chart/operator/templates/rbac.yaml [hunk 2]
```diff
@@ -459,7 +459,7 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
-  name: {{ include "operator.fullname" . | trunc (int (sub 63 (len "-leader-election-role"))) | printf "%s-leader-election-role" }} binding
+  name: {{ include "operator.fullname" . | trunc (int (sub 63 (len "-leader-election-role"))) | printf "%s-leader-election-role" }}binding
```
Comment (kezhenxu94) [inline chart/operator/templates/rbac.yaml:462]:
```suggestion
  name: {{ include "operator.fullname" . | trunc (int (sub 63 (len "-leader-election-role"))) | printf "%s-leader-election-role" }}-binding
```

**Review by kezhenxu94 — APPROVED** [review]
LGTM

File: chart/operator/templates/rbac.yaml [hunk 3]
```diff
@@ -459,7 +459,7 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
-  name: {{ include "operator.fullname" . | trunc (int (sub 63 (len "-leader-election-role"))) | printf "%s-leader-election-role" }} binding
```
Comment (kezhenxu94) [inline chart/operator/templates/rbac.yaml:462]: > Could you describe the issue and your use case?

Kubernetes resource name cannot have spaces and here is space

**Pull request merged**